sdks/python/setup.py

```diff
@@ -169,6 +170,7 @@ def get_version():
     'pytest>=4.4.0,<5.0',
     'pytest-xdist>=1.29.0,<2',
     'pytest-timeout>=1.3.3,<2',
+    'scikit-learn>=0.24.2',
```

Comment:
How does this affect `scikit-learn` in the [base_image_requirements_manual.txt](https://github.com/apache/beam/blob/master/sdks/python/container/base_image_requirements_manual.txt#L42)?

Reply:
AFAIK this is just for testing, like unit and e2e tests.

Reply:
It would be good to get an answer on the above (and similarly for joblib)